metadata-ingestion-modules/airflow-plugin/pyproject.toml

```diff
@@ -29,30 +37,21 @@ classes = ["typing"]
 
 [tool.ruff.lint]
 select = [
```

Comment:
btw we should be using extend-select everywhere instead of `select`

https://docs.astral.sh/ruff/settings/#lint_extend-select